client/src/components/AdvCalendar.js

```diff
@@ -1,52 +1,60 @@
-import { useEffect, useState } from 'react';
-
 import styled from 'styled-components/macro';
-import PropTypes from 'prop-types';
-
+import { useState } from 'react';
 import AdventModal from './AdventModal';
 import CalBackground from '../assets/cal_background.svg';
 import Data from '../data.json';
 
-export default function AdvCal(props) {
-  //Set the month for testing (0 == January, 11 == December)
-  const month = 5;
-  const today = new Date();
+export default function AdvCal() {
+  const [open, setOpen] = useState(false);
```

Comment:
maybe rename to  `openModal, setOpenModal` or `modalOnScreen, setModalOnScreen` or  `renderModal, setRenderModal` so that it's clear what happens with `open` here